### PR TITLE
DOCSP-28796 Compaction page for Node.js

### PR DESCRIPTION
**Pull request by lindseymoore**
### Jira

- https://jira.mongodb.org/browse/DOCSP-28796

### Staged Changes

- [Reduce a Realm File Size - Node.js](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-28796/sdk/node/realm-files/compact-realm/)
- [Reduce a Realm File Size - .NET](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-28796/sdk/dotnet/realm-files/compact-realm/)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [ ] Create Jira ticket for corresponding docs-app-services update(s), if any
- [ ] Checked/updated Admin API
- [ ] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by krollins-mdb — COMMENTED** [review]
@lindseymoore, great start! I think we're pretty close, but there are a couple of things to look at. Let me know if you have questions.

File: examples/node/Examples/compacting.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import Realm from "realm";
+
+describe("test compaction on launch", () => {
+  test("shouldCompact", async () => {
```
[the remark this answers is not in the document]
Comment (lindseymoore) [reply examples/node/Examples/compacting.js:4]: Is it necessary? Similar tests on the compaction pages for the other SDKs don't check that the realm compacts at a certain size. I can make a separate ticket for the test if you think it necessary!

[the remark this answers is not in the document]
Comment (krollins-mdb) [reply examples/node/Examples/compacting.js:4]: Yeah, please make a ticket. The other tests are also taking a shortcut.

**Review by krollins-mdb — APPROVED** [review]
LGTM. Thanks for adding this page!

**Pull request merged**